test/data/filenames_and_counts.tsv

```diff
@@ -1,3 +1,4 @@
 file1	1000
 file2	2000
+
```

Comment:
Why this?

This reminds me that we should probably skip lines only containing `\n` when parsing files.

Reply:
> This reminds me that we should probably skip lines only containing \n when parsing files.

That's why. I skip lines containing only `\n` and now I also test it.